intdataplane/int_dataplane.go

```diff
@@ -558,14 +572,15 @@ func (d *InternalDataplane) apply() {
 			// Queue a resync on the next Apply().
 			r.QueueResync()
 		}
-		for _, t := range d.allIptablesTables {
-			t.InvalidateDataplaneCache()
-		}
```

Comment:
As discussed - the point here is that we still need the dataplane-level force-refresh concept for the route table.